net/http/inc/TCivetweb.h

```diff
@@ -32,6 +35,8 @@ class TCivetweb : public THttpEngine {
 
    Bool_t IsDebugMode() const { return fDebug; }
 
+   Bool_t IsShutdown() const { return fShutdown; }
```

Comment:
Is this `IsShutdown` or `IsShuttingDown`? Maybe `IsTerminating` is more readable?

Reply:
`IsTerminating` probably better - I will push additional commits.
 Actually, all this flags required while I cannot request such flag from `gApplication`